docs/topics/proc_committing-and-pushing-changes-to-the-github-repository-of-your-booster-with-visual-studio-code.adoc

```diff
@@ -0,0 +1,20 @@
+
+[id='committing-and-pushing-changes-to-the-github-repository-of-your-booster-with-visual-studio-code_{context}']
+= Committing and pushing changes to the GitHub repository of your booster with Visual Studio Code
+
+Visual Studio Code provides tooling for committing and pushing code through Git directly in the IDE. Alternatively, you can use Visual Studio Code to make changes and push those changes with the Git CLI.
+
+[discrete]
+== Prerequisites
+
+. All files with changes that you intend to commit are saved.
+
+[discrete]
+== Procedure
+
+. In Visual Studio Code, on the left side, click the _Source Control_ icon.
+. Under _Changes_ click the _+_ icon for each file you want to stage. Alternatively, you can click on the _+_ icon next to _Changes_ to stage all files.
+. In the field above _Staged Changes_ enter a commit message and click the check icon to commit your changes.
+. Click on the _..._ icon and select _Push_ to push your changes.
+. Observe your changes are now available in your GitHub repository.
```

Comment:
From a purely stylistic PoV, I believe this should be part of the previous item because it pertains to simply observing the result. However, I'm not going to dwell on this.